### DISCUSSION TITLE
blog/%EB%94%94%EC%9E%90%EC%9D%B87%EA%B0%80%EC%A7%80

### DISCUSSION THREAD

**Discussion by giscus** [Samdasoo1076/blog#6] [comment]
# blog/%EB%94%94%EC%9E%90%EC%9D%B87%EA%B0%80%EC%A7%80

7 Practical Tips for Cheating at Design

https://blog-samdasoo.vercel.app/blog/%EB%94%94%EC%9E%90%EC%9D%B87%EA%B0%80%EC%A7%80

**Comment by Samdasoo1076** [comment]
올 ㅋ 굿
